#
# Submitted-By: Marc Kleine-Budde, 2006-02-28
# Committed-By: Marc Kleine-Budde
#
# Error:
#
# if arm-softfloat-linux-gnu-gcc -DHAVE_CONFIG_H
# -I. -I. -I. -I./include -I./include -D_REENTRANT -g -O2 -pedantic -W
# -Wformat -Wunused -Wimplicit -Wreturn-type -Wswitch -Wcomment
# -Wtrigraphs -Wformat -Wchar-subscripts -Wuninitialized -Wparentheses
# -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings
# -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes
# -Wnested-externs -Winline -Wredundant-decls -MT runtest.o -MD -MP
# -MF ".deps/runtest.Tpo" -c -o runtest.o runtest.c; \ 
# then mv -f ".deps/runtest.Tpo" ".deps/runtest.Po"; else rm -f
# ".deps/runtest.Tpo"; exit 1; fi runtest.c:4182: error:
# 'streamParseTest' undeclared here (not in a function) make[3]: ***
# [runtest.o] Error 1
#
# Description: 
#
# try to compile a test, thats disabled by config
#
# State:
# 
# not commited, but should be :)
#
Index: libxml2-2.6.23/runtest.c
===================================================================
--- libxml2-2.6.23.orig/runtest.c
+++ libxml2-2.6.23/runtest.c
@@ -4178,6 +4178,7 @@ testDesc testDescriptions[] = {
       errParseTest, "./test/XInclude/docs/*", "result/XInclude/", "", NULL,
       /* Ignore errors at this point ".err", */
       XML_PARSE_XINCLUDE },
+#ifdef LIBXML_READER_ENABLED
     { "XInclude xmlReader regression tests",
       streamParseTest, "./test/XInclude/docs/*", "result/XInclude/", ".rdr",
       /* Ignore errors at this point ".err", */
@@ -4191,6 +4192,7 @@ testDesc testDescriptions[] = {
       /* Ignore errors at this point ".err", */
       NULL, XML_PARSE_XINCLUDE | XML_PARSE_NOXINCNODE },
 #endif
+#endif
 #ifdef LIBXML_XPATH_ENABLED
 #ifdef LIBXML_DEBUG_ENABLED
     { "XPath expressions regression tests" ,
